app/static/js/fields.js

```diff
@@ -0,0 +1,74 @@
+/*
+	Author: ASU CAPSTONE TEAM 2018
+	Date: 03.19.2019
+	Description: Controller for Handling UI data binding and REST request
+	             for Fields
+*/
+
+var app = angular.module('solarsenseApp', []);
+
+app.config(['$interpolateProvider', function($interpolateProvider) {
+  	$interpolateProvider.startSymbol('{a');
+  	$interpolateProvider.endSymbol('a}');
+}]);
+
+app.controller('LinkCtrl', function($scope, $window) {
```

Comment:
The navigation bar global controller has been pulled into master now, so make sure to remove this controller from this file.

Reply:
This has been resolved with merging the change in dev into this branch